test/integration/http-service.yaml.tmpl

```diff
@@ -42,15 +42,15 @@ spec:
       - name: echo
         image: gcr.io/google_containers/echoserver:1.4
         ports:
-        - containerPort: 8080
+        - containerPort: $PORT
       - name: proxy
-        image: gcr.io/istio-test/runtime:experiment
+        image: gcr.io/istio-test/runtime:$TAG
         securityContext:
           runAsUser: 1337
         args:
           - proxy
           - -s
-          - manager:8080
+          - m:8080
```

Comment:
Manager port is fixed. I'm checking port cases for actual services (a and b here)